README.md

```diff
@@ -109,6 +109,17 @@ Finished in 0.024 second
 
 Learn more: [Migrations](docs/migrations.md)
 
+### 6. Monitoring
+Redcord reports metrics to a tracer (for example, [Datadog APM](https://docs.datadoghq.com/tracing/setup/ruby/#manual-instrumentation)) if it is configured.
```

Comment:
I would include something that makes the flexibility here more obvious. Eg. "The tracer any object with a `.trace(.....)` method". And then include datadog as an example.